proto/data.go

```diff
@@ -482,55 +482,9 @@ func NewScanMetadata(nowNanos int64) *ScanMetadata {
 	return &ScanMetadata{
 		LastScanNanos:     nowNanos,
 		OldestIntentNanos: gogoproto.Int64(nowNanos),
-		GC: GCMetadata{
-			ByteCounts: make([]int64, 10),
-		},
 	}
 }
 
-// EstimatedBytes computes the estimated count of bytes which a GC run
```

Comment:
I removed this earlier attempt at estimating need for GC on a range in favor of the gc'able bytes age statistic, which is more symmetrical with the intent age statistic and I think will work better overall at determining when a range should be GC'd.